src/tests/libecalc/core/consumers/test_crossover.py

```diff
@@ -1,62 +1,96 @@
+from datetime import datetime
+from typing import List
+
 import numpy as np
 import pytest
+from libecalc.common.stream import Stream
+from libecalc.common.units import Unit
+from libecalc.common.utils.rates import TimeSeriesFloat, TimeSeriesRate
 from libecalc.core.consumers.consumer_system import ConsumerSystem
 
 
+def create_stream_from_rate(rate: List[float]) -> Stream:
+    timesteps = [datetime(2020, 1, i + 1) for i in range(len(rate))]
+    return Stream(
+        rate=TimeSeriesRate(
+            timesteps=timesteps,
+            values=rate,
+            unit=Unit.STANDARD_CUBIC_METER_PER_DAY,
+        ),
+        pressure=TimeSeriesFloat(
+            timesteps=timesteps,
+            values=[50] * len(rate),
+            unit=Unit.BARA,
+        ),
+    )
+
+
 class TestCrossover:
-    parameterized_crossover_rates = [
-        ([], [], [], []),  # All empty
```

Comment:
No inlet streams should be handled elsewhere, right?

Reply:
If so, then the docstring should say that that is assumed and not checked and handled in that method. Can it be handled? Probably not. ..